packages/block-editor/src/components/media-placeholder/index.js

```diff
@@ -159,6 +159,7 @@ export class MediaPlaceholder extends Component {
 			notices,
 			onSelectURL,
 			mediaUpload,
+			append,
```

Comment:
This component is very tied to the Placeholder would it make sense to pass props not explicitly handler by the media placeholder to the placeholder component?

Reply:
yes we can do the same with `icon` & `notices` as well.

```
...props
```